src/app/Counters/DeleteCounter.tsx

```diff
@@ -41,7 +42,7 @@ const DeleteCounter = (props: {
         >
           {t('cache-managers.counters.delete-action')}
         </Button>,
-        <Button key={'Cancel'} aria-label={'Cancel'} variant={ButtonVariant.link} onClick={props.closeModal}>
+        <Button key={'Cancel'} aria-label={'Cancel'} variant={ButtonVariant.link} onClick={props.closeModal} data-cy="cancelButton">
```

Comment:
I'm wondering if can't use the Key or aria label or other instead of data-cy for this tests?
Maybe also naming more explicitely "cancelButtonCreateCounter" or something like that will be better. Since cancelButton is very generic